doc/Type/Slip.pod6

```diff
@@ -36,7 +36,7 @@ it behaves closer to the C<.Slip> method than the C<slip> subroutine.
 =for code
 my $l = (1, 2, 3);
 say (1, slip 2, 3).raku;  # says (1, 2, 3)      , slips 2, 3 into (1, …)
-say (0, slip $l).raku;    # says (0, $(1, 2, 3)), $l does not break apart
+say (0, slip $l).raku;    # says (0, 1, 2, 3)   , slips from $l into (0, …)
```

Comment:
Thanks for fixing this, but the problem is that whatever it was trying to illustrate will be left orphan. A more integral approach, looking at the context, would be necessary, illustrating how itemizing will escape slipping (if that's what this particular line was aiming at)

Reply:
I've added an example that works the way the original writer wanted as well as the fixed example